enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2495,7 +2495,7 @@ func configureGlobalURLRewrites(ctx context.Context) error {
 	// Discard any existing rewrites initially, then append.
 	flag := "--replace-all"
 	for original, replacement := range map[string]string{
-		"ssh://git@github.com:": "https://github.com/",
+		"ssh://git@github.com/": "https://github.com/",
```

Comment:
Leaving the server port blank seems to work for me:

```
git clone ssh://git@github.com:/buildbuddy-io/buildbuddy /tmp/buildbuddy.git
```

Should we add a new entry instead of overwriting the existing one?

Reply:
Makes sense. I think it's relatively rare to specify a SSH port with `github.com` because it's always 22. But there's no harm in adding it.